sportngin/ical_importer
# SITE-1812-Replace-rical-with-icalendar

Description and Impact
----------------------
> Replaces rical gem with icalendar gem

Deploy Plan
-----------
> N/A

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> Links to bug tickets or user stories.
* [Jira issue SITE-3700](https://sportngin.atlassian.net/browse/SITE-3700)

QA Plan
-------
> Fill in scenarios below. Review [QA Best Practices](https://source.sportngin.com/page/show/1227795-qa-best-practices).


See https://github.com/sportngin/ngin/pull/6757
